lib/comfortable_mexican_sofa/extensions/is_taggable.rb

```diff
@@ -82,9 +82,8 @@ def unkeyword!(keyword_ids:)
       # Synchorize the taggings of this item with the ones in its mirrors. So all of them
       # are associated to the same tags
       def sync_mirrors_with_taggings
-        if is_mirrored_with_taggings
-          mirrors.each { |mirror| mirror.assign_new_keywords!(new_keyword_ids: keywords.map(&:id)) }
-        end
+        return unless is_mirrored_with_taggings
+        mirrors.each { |mirror| mirror.assign_new_keywords!(new_keyword_ids: keywords.map(&:id)) }
```

Comment:
It's part of a boolean expression - remember predicates don't have to return booleans, they can return truthy or falsey, which could be `nil` or not nil. This being perfectly cromulent.